test-cli/src/bin/main.rs

```diff
@@ -123,6 +123,7 @@ fn main() {
                             0,
                             counter_amount,
                             &mut rng,
+                            None,
```

Comment:
Passing `None` here means the test utility no longer generates SCIs that can get accepted by the server. I think a potential for solution for this would be for the test-cli to accept a ledger file and append to it similar to what the tests are doing. That unfortunately does mean that all servers have to share the same ledger file when running this locally. It's acceptable for now, since I think we could use the liquidity bot I am working on for submitting SCIs out of real accounts.
2)